core/trino-main/src/main/java/io/trino/sql/planner/planprinter/NodeRepresentation.java

```diff
@@ -181,13 +185,12 @@ public List<PlanNodeStatsAndCostSummary> getEstimates(TypeProvider typeProvider)
     public static class TypedSymbol
     {
         private final Symbol symbol;
-        private final String type;
+        private final Type actualType;
```

Comment:
why `actualType` and not `type`?
is there any other type in play?

Reply:
+1